docs/user-guide/api-mediation/using-multi-factor-authentication.md

```diff
@@ -0,0 +1,42 @@
+
+Beginning with Zowe version 1.8.0, the Zowe App Framework, Desktop, and all apps present in the SMP/E or convenience builds support out-of-band MFA. Users are required to enter an MFA assigned token or passcode into the password field of the Desktop login screen. Alternatively, users can access the app-server `/auth` REST API endpoint.
+
+When using MFA with Zowe CLI or the API ML Catalog, users are required to log in with their mainframe user name and MFA token.
+
+## Prerequisite
+
+Ensure that you meet the following prerequisite to use MFA with Zowe CLI or API ML Catalog:
+
+* z/OSMF APAR for MFA must be installed on the system. For more information, see [this APAR](https://www.ibm.com/support/pages/apar/PH39582) in IBM Support.
+
+## Known Limitations and Recommendations
+
+* **Unintentional Reuse of MFA Token**
+When z/OSMF is used as a security provider, it is possible to reuse MFA tokens, whereby it is possible to receive a JWT token based on previously used MFA token. This presents a security risk.  
+
+This issue can be resolved by configuring z/OSMF to work properly with API ML.
+Update the z/OSMF configuration with the following parameter:
+`allowBasicAuthLookup="false"`
+
+After applying this change, each authentication call results in generating a new JWT.
+For more information, see [Configuring z/OSMF](./systemrequirements-zosmf/#configuring-zosmf-to-properly-work-with-api-ml) to properly work with API ML, and [Multi-factor authentication configuration](./mvd-configuration/#multi-factor-authentication-configuration) in Configuring Zowe Application Framework.
+
+* No Notification when Additional Input is Required
+Neither Zowe CLI nor API Catalog issue a notification when a user is required to provide additional input. This can occur in cases such as when a user signon attempt triggers the requirements of a **New Pin** or **Next Token**. The user must resolve this situation outside of Zowe. Depending on the current authentication factor enabled (RSA SecurID or RADIUS), the user can use TSO console or MFA Self-service facilities.
+
+We recommend you first try to access self-service facilities and resolve the issue there. If you are unable to access your self-service facilities, contact your system administrator.
+
+:::tip
+* For more information about how to manage multi-factor authentication credentials in AAM, see [Manage Multi-Factor Authentication Credentials (IBM RACF)](https://techdocs.broadcom.com/us/en/ca-mainframe-software/security/ca-advanced-authentication-mainframe/2-0/using-with-ibm-racf/manage-multi-factor-authentication-credentials-ibm-racf.html) in the Advanced Authentication Mainframe 2.0 Broadcom documentation.
+* For more information about how to resolve the RADIUS Access Challenge, see the sub-topic RADIUS Access Challenge Considerations in the previously referenced documenation.
+:::
+
+* Token Expiration when Stored in the Authorization Dialog in "Try it out"
```

Comment:
Add tip - For more information about how to manage multi-factor authentication credentials in IBM Z MFA, see
[IBM Z Multi-Factor Authentication](https://www.ibm.com/products/ibm-multifactor-authentication-for-zos).

Additionally, Zowe API ML can be configured to accept OIDC/OAuth2 user authentication tokens. In this particular case, MFA support is built into the OIDC provider system. It does not rely on the mainframe MFA technology, but is equally secure.